client/src/components/settings/Certification.js

```diff
@@ -190,14 +190,16 @@ export class CertificationSettings extends Component {
             >
               Front End
             </MenuItem>
-            <MenuItem
-              bsStyle='primary'
-              href={githubLink}
-              rel='noopener noreferrer'
-              target='_blank'
-            >
-              Back End
-            </MenuItem>
+            {githubLink !== 'empty' && (
```

Comment:
I do not like that we would have to keep a comparison with `'empty'`? How can we make this cleaner? Should we be able to simply check against `''`?

Reply:
I tried an empty string and null, but I was getting errors. I can take a deeper look.

Reply:
Can you elaborate on the errors when you get a chance? Thanks.

Reply:
I guess the error wasn't with the comparison here, It was when trying to send those values (an empty string or null) to the backend. Meaning when I set it as one of those in the completion-epic, the `github` value wouldn't be found I guess, and throw back a server error. "Cannot reach the server to update your progress" is what the flash message says I think; it disappears quickly.

So are you concerned with the comparison here, or storing the string `empty` in the db?

Reply:
> So are you concerned with the comparison here, or storing the string `empty` in the db?

Yes, I am concerned about the db values. I would imagine we would want to have a standard value instead of the string `empty`, which could get lost in time down the lane, leading to confusion.

Reply:
What about just not storing anything?  The api-server currently rejects challenges with no `githubLink`, so that would have to change, but I think it's fine to omit keys from documents.

As long as everything that looks for them can cope with them being missing, it should be okay.  I'm not a Mongo expert, though.

Reply:
> As long as everything that looks for them can cope with them being missing, it should be okay.

Yes, it should be. Ideally we can store `null` and update the api to not reject empty values as it does currently for GitHub.